oracle/tests/test_instant_client.py

```diff
@@ -39,50 +39,50 @@
         ),
     ],
 )
-def test__get_connection_instant_client(instance, dd_run_check, aggregator, expected_tags):
+def test__get_connection_oracledb(instance, dd_run_check, aggregator, expected_tags):
     """
     Test the _get_connection method using the instant client
     """
     check = Oracle(CHECK_NAME, {}, [instance])
-    check.use_oracle_client = True
+    check.use_jdbc = mock.Mock(return_value=False)
```

Comment:
we should not need to mock anything here, just not define the jdbc path on the instance